### PR TITLE
Meeting summary 2016-11-17

### PR DESCRIPTION
**Pull request by G1lius**

### CODE REVIEW

File: _posts/en/meetings/2016-11-17-meeting.md [hunk 1]
```diff
@@ -0,0 +1,146 @@
...
+
+## Priority removal
+
+### background
+
+The priority system used to assign transactions with a priority based on age, size, and number of inputs, which made some transactions free. This has a large codebase and efforts have been made to remove the system as it can't be expected from miners to keep including 0-fee transactions.
+
+### meeting comments
+
+Morcos notes the priority code doesn't serve any function anymore, perhaps a target can be set to remove all the priority code for 0.15. There's a lot of stuff merged on the way for priority already.
```
Comment (instagibbs) [inline _posts/en/meetings/2016-11-17-meeting.md]: Might want to note that it was noted that Luke-jr likely disagrees with this statement.

**Comment by btcdrak** [comment]
Some travis failures:

```
- ./_site/en/meetings/2016/11/17/index.html
  *  internally linking to en/meetings/2016/07/14/#notes--short-topics, which does not exist (line 428)
  *  internally linking to en/meetings/2016/11/10/#hybrid-spv, which does not exist (line 447)
  *  trying to find hash of en/meetings/2016/07/14/#notes--short-topics, but /home/travis/build/bitcoin-core/bitcoincore.org/_site/en/meetings/2016/11/17/en/meetings/2016/07/14 does not exist (line 428)
  *  trying to find hash of en/meetings/2016/11/10/#hybrid-spv, but /home/travis/build/bitcoin-core/bitcoincore.org/_site/en/meetings/2016/11/17/en/meetings/2016/11/10 does not exist (line 447)
htmlproof 2.6.4 | Error:  HTML-Proofer found 4 failures!
```

File: _posts/en/meetings/2016-11-17-meeting.md [hunk 2]
```diff
@@ -0,0 +1,146 @@
...
+
+## Account removal
+
+### background
+
+Currently bitcoin-core uses a system of accounts. Third parties using this can have multiple problems with this, and it's been the consensus [for a long time](https://github.com/bitcoin/bitcoin/issues/3816) the system should be removed and replaced by a 'label' system.
+
+### meeting comments
+
+Earlier this year Wumpus opened a [pull request][#7729] to introduce a 'label' API for the wallet (previously mentioned in the [2016-07-14 meeting](en/meetings/2016/07/14/#notes--short-topics)). It still hasn't received much review.
```
Comment (btcdrak) [inline _posts/en/meetings/2016-11-17-meeting.md]: looks like the link is missing a leading `/` -> `/en/meetings/2016/07/14/#notes--short-topics`

File: _posts/en/meetings/2016-11-17-meeting.md [hunk 3]
```diff
@@ -0,0 +1,146 @@
...
+### meeting conclusion
+
+- Review [#7729][] (rpc: introduce 'label' API for wallet)
+- ping dooglus for user feedback
+
+## Auxiliary block requests
+
+### background
+
+Jonasschnelli opened a [pull request][#9171] to introduce auxiliary block requests (previously "Out-of-band Block Requests"). This function allows to request blocks, if available on disk, and if not will be downloaded and prioritized over normal IBD downloads. This change is needed to run the SPV wallet, talked about in [last weeks meeting](en/meetings/2016/11/10/#hybrid-spv).
```
Comment (btcdrak) [inline _posts/en/meetings/2016-11-17-meeting.md]: link missing leading `/` -> `/en/meetings/2016/11/10/#hybrid-spv`

**Pull request merged**